components/Opportunities/index.jsx

```diff
@@ -0,0 +1,105 @@
+import Image from 'next/image';
+import PropTypes from 'prop-types';
+import {
+  Button, Card, Col, Row, Typography,
+} from 'antd';
+import styled from 'styled-components';
+import opportunities from './opportunities.json';
+
+const LAUNCH_CONTACT_URL = 'https://t.me/pahlmeyer';
+
+const StyledCard = styled(Card)`
+  width: 100%;
+  height: 100%;
+  align-items: center;
+  .ant-card-body {
+    min-height: 100%;
+    display: flex;
+  }
+  margin-bottom: 24px;
+`;
+
+const OpportunityCard = ({
+  agentName,
+  agentDescription,
+  project,
+  image,
+  background,
+}) => (
+  <StyledCard>
+    <Row style={{ columnGap: '12px', width: '100%', minHeight: '100%' }}>
+      <Col xs={24} md={14}>
+        <h2>{agentName}</h2>
+        <p style={{ whiteSpace: 'pre-line' }}>
```

Comment:
better to avoid using style attribute if possible, here it's fine as get to simple, but usually it impacts performance because [it causes unnecessary re-renders ](https://www.linkedin.com/pulse/stop-using-inline-styles-react-js-azeem-aleem)

Reply:
What would be the better way to do this?